root/io/io/CMakeLists.txt

```diff
@@ -4,3 +4,5 @@ RB_ADD_GBENCHMARK(IOBenchmarks
   Core RIO Tree TreePlayer
   )
  
+ add_test(NAME rootbench-IOBenchmarks
```

Comment:
We should hide that behind `RB_ADD_GBENCHMARK` in `AddRootBench.cmake`

Reply:
ok will be done!